consoles/s3270.pm

```diff
@@ -246,7 +246,10 @@ sub expect_3270() {
                     push @$result, $line;
                 }
 
-                confess "status line matches neither buffer_ready nor buffer_full:\n" . Dumper($result) . $status_line;
+                # TODO the following should be "confess" but since
+                # 7a2e1b41 confess itself dies inside the die handler w/o
+                # helpful output
+                warn "status line matches neither buffer_ready nor buffer_full:\n" . Dumper($result) . $status_line;
```

Comment:
removed the TODO comment. just `warn` should be ok (discussed with coolo), fatal is not necessary here